tests/src/test/scala/common/Wsk.scala

```diff
@@ -922,6 +928,10 @@ sealed trait RunWskCmd extends Matchers {
         jsonStr.substring(jsonStr.indexOf("\n") + 1).parseJson.asJsObject // Skip optional status line before parsing
     }
 
+    def parseJsonValueString(jsonStr: String): JsValue = {
```

Comment:
dont we have this method somewhere else? vaguely recall you added something like it recently.